app/models/privacy_policy.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+class PrivacyPolicy < ApplicationRecord
+  class Acceptance < ApplicationRecord
+    self.table_name = :privacy_policy_acceptances
+
+    belongs_to :user
+    belongs_to :privacy_policy
+  end
+
+  def self.current
+    order(:major_version, :minor_version).last
+  end
+
+  def acceptance_required?(user)
+    return false if !user || user.admin?
+    return false unless user.induction_coordinator_profile
+
+    !Acceptance
+      .joins(:privacy_policy)
+      .where(user: user)
+      .where("privacy_policies.major_version >= ?", major_version)
+      .exists?
```

Comment:
Instead of !exists why not just Acceptance.joins...... .none?

Reply:
Awesome, it seems `none?` is also executing `EXIST` query (I was under impression it will load all the records and delegate `none?` to the @results array)